### PR TITLE
Broadphase fix take 2

### PR DESCRIPTION
**Pull request by metalgearsloth**
Original PR https://github.com/space-wizards/RobustToolbox/pull/2318

### CODE REVIEW

File: Robust.Shared/Physics/FixtureSystem.cs [hunk 1]
```diff
@@ -234,8 +240,8 @@ public void DestroyFixture(PhysicsComponent body, Fixture fixture, bool updates
 
         private void OnPhysicsShutdown(EntityUid uid, PhysicsComponent component, ComponentShutdown args)
         {
-            if (EntityManager.GetComponent<MetaDataComponent>(uid).EntityLifeStage > EntityLifeStage.MapInitialized) return;
-            EntityManager.RemoveComponent<FixturesComponent>(uid);
+            if (MetaData(uid).EntityLifeStage > EntityLifeStage.MapInitialized) return;
+                EntityManager.RemoveComponent<FixturesComponent>(uid);
```
Comment (PaulRitter) [inline Robust.Shared/Physics/FixtureSystem.cs]:
```suggestion
            EntityManager.RemoveComponent<FixturesComponent>(uid);
```

File: Robust.Shared/GameObjects/Systems/SharedPhysicsSystem.Bodies.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    [Dependency] private readonly IMapManager _mapManager = default!;
+
+    private void OnPhysicsInit(EntityUid uid, PhysicsComponent component, ComponentInit args)
+    {
+        if (component.BodyType == BodyType.Static)
+        {
+            component._awake = false;
+        }
+
+        var xform = EntityManager.GetComponent<TransformComponent>(uid);
```
Comment (PaulRitter) [inline Robust.Shared/GameObjects/Systems/SharedPhysicsSystem.Bodies.cs]:
```suggestion
        var xform = Transform(uid);
```

**Review by PaulRitter — COMMENTED** [review]
partial review

**Comment by metalgearsloth** [comment]
FindContacts exception is on master since IEntity refactor anyway so no known issues.

**Pull request merged**